pyproject.toml

```
@@ -0,0 +1,56 @@
+[build-system]
+requires = ["setuptools>=61.0", "pyasyncore", "cryptography", "pyoslog"] # core requirements are needed for version detection, which requires importing the script
```

With #201 pyoslog is only used when the logger is initialized, so this can be dropped too.

```suggestion
requires = ["setuptools>=61.0", "pyasyncore", "cryptography"] # core requirements are needed for version detection, which requires importing the script
```

---

Thanks! I'll take a look at #201 when I get chance. You're correct about pyasyncore: only after Python 3.12.